docs/service_controller.md

```diff
@@ -4,32 +4,32 @@ The service controller is responsible for watch for service and node object chan
+| service.beta.kubernetes.io/aws-load-balancer-healthcheck-unhealthy-threshold   | [2-10]                              | 2   | ELB,NLB   | The number of consecutive failed health checks that must occur before declaring an EC2 instance unhealthy. |
+| service.beta.kubernetes.io/aws-load-balancer-internal                          | [true\|false]                       | -   | ELB,NLB   | Indicates that the load balancer should be internal. |
+| service.beta.kubernetes.io/aws-load-balancer-proxy-protocol                    | [*]                                 | -   | ELB       | Enables the proxy protocol on an ELB. Right now we only accept the value "*" which means enable the proxy protocol on all ELB backends. In the future we could adjust this to allow setting the proxy protocol only on certain backends. |
+| service.beta.kubernetes.io/aws-load-balancer-ssl-cert                          | IAM or ACM ARN                      | -   | ELB,NLB   | Requests a secure listener. Value is a valid certificate ARN. For more, see the [elb listener config guide](http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/elb-listener-config.html).  CertARN is an IAM or CM certificate ARN. |
+| service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy            | -                                   | ELBSecurityPolicy-2016-08 | ELB,NLB   | Specifies SSL negotiation settings for the HTTPS/SSL listeners of your load balancer. Defaults to the default ELB policy. |
+| service.beta.kubernetes.io/aws-load-balancer-ssl-ports                         | Comma-separated list                | *   | ELB,NLB   | Specifies a comma-separated list of ports that will use SSL/HTTPS listeners. Defaults to all. |
+| service.beta.kubernetes.io/aws-load-balancer-type                              | [nlb]                               | -   | ELB,NLB   | Indicates the type of Load Balancer. The only valid value is nlb.  Leaving this field blank is equivalent to selecting ELB. |
+| service.beta.kubernetes.io/aws-load-balancer-eip-allocations                   | Comma-separated list                | -   | NLB       | List of EIP allocations to associate with a internet-facing load balancer. Only valid for NLB. |
+| service.beta.kubernetes.io/aws-load-balancer-healthcheck-path                  | -                                   | /   | ELB,NLB   | Specifies the http path for the health check in case of http/https protocol. |
+| service.beta.kubernetes.io/aws-load-balancer-healthcheck-port                  | [traffic-port\|1-65535]             | traffic-port | ELB,NLB   | Specifies the TCP target port for the target group health check. |
+| service.beta.kubernetes.io/aws-load-balancer-healthcheck-protocol              | [tcp\|http\|https]                  | tcp | ELB,NLB   | Specifies the protocol to use for the target group health check. |
```

Comment:
Good catch! What do you think to document the current state of master, upper case requirement for ELB (as described in https://github.com/kubernetes/cloud-provider-aws/issues/1181), while https://github.com/kubernetes/cloud-provider-aws/pull/1182 is not merged yet? Or do you want to wait for it?